Modules/pyexpat.c

```diff
@@ -1635,6 +1582,32 @@ static struct PyModuleDef pyexpatmodule = {
         NULL
 };
 
+static int init_handler_descrs(void)
```

Comment:
This function looks _really_ low-level. Isn't there a more "official" API way to do this?
And why is this called after `PyType_Ready()` and does not invalidate the type/method cache?